devicetypes/iblinds/iblinds-zwave-v3.src/iblinds-zwave-v3.groovy

```diff
@@ -0,0 +1,358 @@
+/**
+ *  Copyright 2020 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  2020-08-31: Force Lifeline association. 
+ *				Added Schedule for daily battery check (HealthCheck /Check Inverval not seeming to fire correctly)
+ *  2020-09-04: Fixed default ON value and preset 
+ */
+import groovy.json.JsonOutput
+
+
+metadata {
+	definition (name: "iblinds Z-Wave v3", namespace: "iblinds", author: "HABHomeIntel", ocfDeviceType: "oic.d.blind",  mnmn: "SmartThings", vid: "generic-shade-3") {
```

Comment:
Since that's a v3 version of previous "iblinds Z-wave" DTH and both of them support this device:
fingerprint mfr:"0287", prod:"0004", model:"0071", deviceJoinName: "iBlinds Window Treatment", 

I suggest to consider refactoring the existing DTH (iblinds-zwave.groovy) instead of creating a new one.

Reply:
Hi, thanks for the reply.  The v3 fingerprint will need to be removed from the current v2 DTH.   v3 supports preferences (Z-Wave Configuration Parameters) that v2 will not support.  So, I believe they will need to be separate DTHs.   I put in a request to have the v3 fingerprint removed from the existing v2 device handler but I have not received a response.  

Please advise.

Reply:
@habhomegit I mentioned this to @dkirker internally, because a migration of currently working devices (to the new DTH) might have to be done. Right after we have the confirmation we will continue the code review.